components/price/price.js

```diff
@@ -100,6 +101,15 @@ function Price(props) {
       )
     : null;
 
+  const asteriskSymbol = '*';
```

Comment:
What's the purpose of creating this?

Reply:
@froskie 
May be in the future we will need another symbol/symbols. :-)
But for now this variable is redundant. I agree.
Should I remove it?